### ISSUE TITLE
Add Sending / Error UI for messages in chat

### ISSUE THREAD

**Issue by covrter** [ephemeraHQ/converse-app#1039]
&gt; when messages are hanging on send, showing a “sending...” indicator could help manage expectations. What do you think?
Absolutely let’s add this UI (and its related error states, etc.) I’ll create a ticket and let’s sync on the states <@U066TU07XT4>

[Slack Message](https://xmtp-labs.slack.com/archives/C07DX271K6V/p1729527515847519?thread_ts=1729457969.850679&cid=C07DX271K6V)

**Comment by alexrisch** [comment]
@covrter Can you help link this figma?

**Comment by covrter** [comment]
Yes! In general we'll be adding functionality to the Message Status area over time.

[Here are example treatments for](https://www.figma.com/design/p6mt4tEDltI4mypD3TIgUk/Converse-App?node-id=3049-5250&t=A0il52VpkGvxlGFq-1)
- sent
- sending (maybe this has some min threshold before appearing)
- no connection (bubble gets a style that shows that it was not sent)

[Here's the design system component](https://www.figma.com/design/PUfpkzt9dFX5GR1U1Fw3mQ/Converse-Global-Styles?node-id=487-12121&t=AtsyCXMHyHAkkGpL-1)

Would plan for this area to accommodate addl states like
- delivered (curious if we can do this today)
- seen
- etc.
